inventory/sample.libvirt.example.com.d/inventory/hosts

```diff
@@ -0,0 +1,44 @@
+[infra]
+cloud-host.local
```

Comment:
Add a comment here to explain that this is the KVM host machine?

Add a comment about possibly adding `ansible_become=true` as a var?